testem.js

```diff
@@ -0,0 +1,13 @@
+/*jshint node:true*/
+module.exports = {
+  "framework": "qunit",
+  "test_page": "tests/index.html?hidepassed",
+  "disable_watching": true,
+  "launch_in_ci": [
+    "PhantomJS"
```

Comment:
You have to copy all the settings from the `testem.json` file. We don't use PhantomJS to test. So please copy the current settings and remove the `testem.json` file